<launch>
    <node pkg="digital_futures" type="marker_pose_estimator" name="marker_pose_estimator" output="screen">
        <param name="camera_calibration" value="$(find digital_futures)/config/calibration.yaml" />
        <param name="dictionary_id" value="0" />
        <param name="corner_refinement_method" value="1" />
        <param name="marker_length" value="0.070" />
        <param name="robot_id" value="robot_1" />
        <param name="output_path" value ="$(find digital_futures)/output/" />
    </node> 
    <node pkg="digital_futures" type="motor_control" name="motor_control" output="screen">
        <param name="robot_id" value="robot_1" />
    </node>
</launch>

<!-- 
Dictionary Enum: 
    DICT_4X4_50=0, DICT_4X4_100=1, DICT_4X4_250=2,
    DICT_4X4_1000=3, DICT_5X5_50=4, DICT_5X5_100=5, DICT_5X5_250=6, DICT_5X5_1000=7,
    DICT_6X6_50=8, DICT_6X6_100=9, DICT_6X6_250=10, DICT_6X6_1000=11, DICT_7X7_50=12,
    DICT_7X7_100=13, DICT_7X7_250=14, DICT_7X7_1000=15, DICT_ARUCO_ORIGINAL = 16,
    DICT_APRILTAG_16h5=17, DICT_APRILTAG_25h9=18, DICT_APRILTAG_36h10=19, DICT_APRILTAG_36h11=20

Corner Refinement Method:
    (0: None, 1: Subpixel, 2:contour, 3: AprilTag 2)
-->